server/src/main/java/com/lunchtime/models/CategoryFood.java

```diff
@@ -3,7 +3,6 @@
 import javax.persistence.*;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.Size;
-
```

Comment:
IDEA has very useful keybindings `Ctrl+Alt+O` and `Ctrl+Alt+L`
This line divided imports from different packages